submit-rules.md

```diff
@@ -47,12 +47,12 @@ Plugins must not list **redundant** API versions in the plugin.yml. Only the ear
 ### 2. No obfuscation
 The source code must be **readable and not obfuscated**. All PHP code can eventually be deobfuscated, so there is really no point of close-sourcing them.
 
-### 3. No eye-catching messages in startup/shutdown
-Colored messages, ASCII art, etc., are not allowed when the plugin is enabling/disabling under normal circumstances. This is explained in https://bit.ly/pmcolors.
+### 3. No useless messages in startup/shutdown
+Plugins must not output unnecessary status messages like "I have been enabled", "Author: Xxx", etc., unless the plugin startup/shutdown takes really much time (more than 1 second) such that the user may become impatient. See https://bit.ly/pmcolors
```

Comment:
It may be better to expand the URL since trust in shortened links is dropping.

Reply:
The shortlink was supposed to be mnemonic. 
![image](https://user-images.githubusercontent.com/19623715/52571328-50d66500-2e50-11e9-8151-06a4efc7804f.png)

Reply:
> The shortlink was supposed to be mnemonic.
> ![image](https://user-images.githubusercontent.com/19623715/52571328-50d66500-2e50-11e9-8151-06a4efc7804f.png)

Maybe add a redirect like `https://pmmp.io/colors`. I would trust the link and it would be memorable.